docs/examples/one_tank.md

```diff
@@ -0,0 +1,219 @@
+# One-Tank Example
+
+This example shows how to train two different models to predict the time-series behavior of a single water tank with a height-dependent outflow and a time-varying inflow.
+It is inspired by the one-tank simulink model used for control design [^1].
+A schematic drawing of the system is shown in the following figure.
+
+![The one-tank system](./images/one_tank.svg)
+
+The complete source code for this example can be found [in the repository](https://github.com/flowcean/flowcean/blob/main/examples/one_tank/run.py).
+See also [Run this example](#run-this-example) on how to run this example locally.
+
+The dynamics of the system can be described by an ordinary differential equation (ODE).
+This type of equation relates the change in a variable (it's derivative) to its actual value.
+
+For this example the system can be descirbed by the ODE
+
+$$ \dot{x} = \frac{b V(t) - a \sqrt{x}}{A} $$
+
+where $x$ is the water level in the tank, $\dot{x}$ is the change of the water level over time, $V(t)$ is the time-dependent inflow, $A$ is the tank area, and $a$ and $b$ are scaling constants for the equation.
+The solution of an ODE is not a single value, but a function (here $x(t)$) or a series of it's values for different times $t$.
+As solving an ODE can be quite complicated, ofthen numerical solvers are used which compute solution points starting from an initial value.
+In this example, the initial value is the initial level of the liquid $x(0) = x_0$ in the tank.
+
+The other free parameters from the above equation are set to
+
+ $A$ | $b$ |  $a$
+-----|-----|------
+ $5$ | $2$ | $0.5$
+
+The inflow is given by $V(t) = \mathrm{max}\left(0, \sin\left( 2 \pi \frac{1}{10} t \right)\right)$ and as the initial condition is $x_0 = 1$.
+Using a suitable numerical solution algorithm, the equation can be solved for the level $x_n$.
+Since the solution is not continuous, the level is not a function of time, but a discrete function of the sample number $n$.
+The corresponding time can be calculated by multiplying the sample number $n$ by the step size $h$ between two samples.
```

Comment:
to me, "T" is more common than "h" for the sampling period. Specifically as "h" may be confused with the height of the tank, here.